source/upgrade.txt

```diff
@@ -76,6 +52,14 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the driver from v4.0 to v4.5, address all breaking changes from
 the version after v4.0 including any listed under v4.5.
 
+.. _java-breaking-changes-v4.8:
+
+Version 4.8 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- The driver ends support for connecting to MongoDB Server versions v3.4 and
+  earlier. To learn more about this change, see the :ref:`<java-server-release-change-v4.8>`
+  section.
```

Comment:
Optional: At the end of this page, you just say, "see <section>" instead of "see the <section> section." Maybe a little cleaner to do that here too.

Reply:
I have a preference similar to described in this previous response. I think it helps the reader identify what the UX will be, and which heading to look under if it doesn't happen to target the exact position on their browser due to how the page gets rendered.

For context, this was something the team had also agreed was a good idea a while back, but open to reasons why we shouldn't do it and changing that. If we decide to discontinue this, we should also ticket the work to standardize across docs.